packages/client/src/tests/mock-offline-data.ts

```diff
@@ -478,7 +478,8 @@ export const mockOfflineData = {
     INFORMANT_SIGNATURE: false,
     INFORMANT_SIGNATURE_REQUIRED: false,
     USER_NOTIFICATION_DELIVERY_METHOD: 'email',
-    INFORMANT_NOTIFICATION_DELIVERY_METHOD: 'email'
+    INFORMANT_NOTIFICATION_DELIVERY_METHOD: 'email',
+    SIGNATURE_REQUIRED_FOR_ROLES: ['LOCAL_REGISTRAR', 'NATIONAL_REGISTRAR']
```

Comment:
@Zangetsu101 just a note for the future but maybe if would be best to start adding user features like this to `User.scope` so for instance now we'd add `scope: ['declare', 'has-signature']` when we create the users. That way, over time, we'd reduce the significance of what a "LOCAL_REGISTRAR" for instance means and countries could basically create any user type they want